doc/source/formats/reference/xtc.rst

```diff
@@ -0,0 +1,22 @@
+.. -*- coding: utf-8 -*-
+.. _XTC-label:
+
+========================================
+XTC (GROMACS compressed trajectory file)
+========================================
+
+.. include:: classes/XTC.txt
+
+The GROMACS XTC trajectory compresses data with reduced precision (3 decimal palces by default). MDAnalysis can only read coordinates from these files. See :ref:`TRR-label` for uncompressed files that provide velocity and force information.
+
+
+Reading in
+==========
+
+MDAnalysis uses XDR based readers for GROMACS formats, which store offsets on the disk. The offsets are used to enable access to random frames efficiently. These offsets will be generated automatically the first time the trajectory is opened, and offsets are generally stored in hidden ``*_offsets.npz`` files. Occasionally, MDAnalysis fails to `read XDR offsets, resulting in a fatal error`_.
```

Comment:
> "Occasionally, MDAnalysis fails to `read XDR offsets, resulting in a fatal error`_."

I would make it a footnote or add it at the end. By having it right in the text it suggests that this is a common thing. Also say that the issue has workarounds.